README.md

```diff
@@ -1,22 +1,61 @@
-# Linking Beans
+# Dependency Injection
 
 ## Learning Goals
 
-- Link beans using methods in the config class.
-- Use the `@Autowired` annotation to create bean links.
+- Manually link or wire beans using methods in the config class.
+- Auto-wire beans using the `@Autowired` annotation.
 
 ## Introduction
 
-We will learn how to create a link between two beans in this lesson. Let’s say
-we have a `Human` instance in our Spring context along with the `Dog` instance.
-We connect them together such that the `Human` instance “has a” `Dog`.
+Now that we understand a little more about beans and the application context,
+let's discuss how we can use these beans and reference them.
+
+In our spring-web-demo project, we referenced the `LunchService` in the
+`LunchController` class like this:
+
+```java
+    // Add a reference to the LunchService
+    private final LunchService lunchService;
+    
+    // Add a constructor
+    @Autowired
+    public LunchController(LunchService lunchService) {
+        this.lunchService = lunchService;
+    }
+```
+
+In that lesson we referred to this as "Spring magic". But this "Spring magic"
+really is something we call dependency injection in action!
+**Dependency injection** is a concept used in the Spring framework to inject
+objects into other objects. In traditional programming, which we have used thus
+far, we would do something like this instead:
+
+```java
+this.lunchService = new LunchService();
+```
+
+But with dependency injection, we don't have to instantiate the `LunchService`.
+Instead, this technique allows us to loosely couple our components and shift the
+work of instantiation onto Spring to manage our beans within the application
+context! In other words, it will look at other beans defined in the context to
+inject the specific field or parameter values.
+
```

Comment:
Many of the BlackRock students were confused about why dependency injection is a good thing. Would it be possible to add more detail about why loose coupling is good? Or why it's easier to write unit tests?